README.md

```diff
@@ -1,26 +1,197 @@
 # Rails Layouts And Templates
 
+This lesson will show you how to use layouts to achieve a common look and feel between different views in your app.
+
 ## Objectives
 
+After this lesson, you'll be able to...
+
 1. Identify the default application layout.
 2. Yield to view templates from a layout.
 3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## Life Without Layouts
+
+Imagine that you're tasked to build an online store app with Rails.
```

Comment:
Love that you're immediately drawing the student in by using the second-person and by putting them in a scenario. Scenario-based learning is super effective.